guides/pricing_and_taxation.md

```diff
@@ -0,0 +1,127 @@
+# Pricing and Taxation
+
+Taxation and prices are two intertwined topics in Solidus, mainly because a large number of jurisdictions in the world use VAT-style taxation, which influences the price at which an item is sold.
+
+## Taxation
+
+Solidus represents taxes for an order's items by using `Spree::TaxCategory` and `Spree::TaxRate`.
+
+Tax rates are associated with a tax category and a zone. If an order's `tax_address` (usually the shipping address) falls within a zone with tax rates, those rates will apply to all line items and shipments with respectively matching tax categories.
+
+When a tax rate gets applied to a line item, two things happen: A tax adjustment is created on the line item, and the line item's `additional_tax_total` and `included_tax_total` are changed to reflect the newly created adjustments. The amounts are then added to the order's `*_tax_total` fields.
+
+### Tax Categories
+
+Products within Solidus can be linked to Tax Categories, which are then used find the taxation rate for the products when they are purchased. One Tax Category can be set to being the default for the entire system, which means that if a product doesn't have a tax category set, then the default tax category would be used.
+
+The Solidus default is to treat everything as exempt from tax. Only if a product has a tax category set, or there is a global default tax category, will products be taxed. The tax category is specific to taxation, and is normally never seen by the user so you could call it something generic like "Taxable Goods." If you wish to tax certain products at different rates, however, then you will want to choose something more descriptive (ex. "Clothing.").
+
+### Tax Rates
+
+A tax rate is essentially a percentage amount charged based on the net price. Tax rates also contain other important information.
```

Comment:
I think the phrase "net price" merits further explanation. Does that include item-level adjustments? Order-level adjustments?